deploy/helm/omni/templates/deployment.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "omni.name" . }}
+  namespace: {{ .Values.omni.namespace.name }}
+  labels:
+    {{- include "omni.labels" . | nindent 4 }}
+    app.kubernetes.io/component: omni
+    {{- with .Values.omni.deployment.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- toYaml .Values.omni.deployment.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.omni.deployment.replicaCount }}
+  selector:
+    matchLabels: {{- include "omni.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+    {{- if .Values.omni.deployment.podAnnotations }}
+      annotations:
+      {{- range $key, $value := .Values.omni.deployment.podAnnotations }}
+        {{ $key }}: {{ $value | quote }}
+      {{- end }}
+    {{- end }}
+      labels: 
+      {{- include "omni.labels" . | nindent 8 }}
+      app.kubernetes.io/component: omni
+      {{- with .Values.omni.deployment.labels }}
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
+    spec:
+      containers:
+      - name: omni
+        image: {{ .Values.omni.deployment.image }}
+        imagePullPolicy: {{ .Values.omni.deployment.imagePullPolicy }}
+        resources:
+          {{- with .Values.omni.resources }}
+          {{- toYaml . | nindent 12 }}
+          {{- end }}
+        ports:
+          - name: api
+            containerPort: {{ .Values.omni.service.api.port }}
+            protocol: TCP
+          - name: siderolink
+            containerPort: {{ .Values.omni.service.siderolink.api.port }}
+            protocol: TCP
+          - name: wireguard
+            containerPort: {{ .Values.omni.service.siderolink.wireguard.port }}
+            protocol: UDP
+          - name: event-sink
+            containerPort: {{ .Values.omni.service.eventSink.port }}
+            protocol: TCP
+          - name: k8s-proxy
+            containerPort: {{ .Values.omni.service.k8sProxy.port }}
+            protocol: TCP
+        env:
+        - name: OMNI_ACCOUNT_ID
```

Comment:
Any reason this is set as an environment variable when everything else is set as args?

Reply:
No, this is no longer relevant. This is a remnant of a different approach I was trying at first, but moved away from. The other env references have all been removed but evidently I forgot to remove it here. Will update.

Reply:
removed